### PR TITLE
fix save/load issue for deepfm

### PR DESCRIPTION
**Pull request by yuanwang2011**
It seems that the deepfm model cannot be loaded successfully after saving, for example
https://github.com/yuanwang2011/DeepCTR/blob/master/examples/run_deepfm_SL.py
I think a reasonable solution is to keep the FM logit as a list till the end, then apply the add_func for all the logits.

### CODE REVIEW

**Comment by shenweichen** [comment]
we have solved this problem in deepctr v0.9.1,please use `pip install -U deepctr` to upgrade latest version.
